README.md

```diff
@@ -21,6 +21,13 @@ npm run dev
 npm run typecheck
 
 # Linting (via eslint)
+npm run lint
+
+# Testing via vitest
+npm run test
+
+# Coverage via vitest, istambul coverage provider
+npm run test:cov
```

Comment:
let's turn on coverage by default!

Reply:
is coverage enforced by default now? I thought to rememvber that one has to turn in on